packages/headless/src/entity/order.ts

```diff
@@ -163,9 +163,16 @@ export class OrderResponseEntity {
   paypalPaymentId?: string;
   paypalRedirectUrl?: string;
 }
-// export class OrderListWithoutUserId {
 
-// }
+//sorting query entity
+export enum SortTypesDto {
+  asc = 'asc',
+  desc = 'desc'
+}
+export class OrderSortQuery {
+  sortField?: string;
```

Comment:
specify some fields, on which you want to provide sorting. don't take anything from front-end.